test/integration/node_pool/controls/gcloud.rb

```diff
@@ -174,6 +174,22 @@
             )
           )
         end
+
+        it "has the expected linux node config sysctls" do
+          expect(data['nodePools']).to include(
+            including(
+              "name" => "pool-01",
+              "config" => including(
+                "linuxNodeConfig" => including(
+                  "sysctls" => including(
+                    "net.core.netdev_max_backlog" => "10000"
```

Comment:
```suggestion
                    "net.core.netdev_max_backlog" => "10000",
```

Reply:
We can probably assert the whole object instead of `including` because there should not be any other keys in `sysctls`.
```ruby
"linuxNodeConfig" => including(
                  "sysctls" => {
                    "net.core.netdev_max_backlog" => "10000",
                    "net.core.rmem_max" => "10000"
                  }
                )
```

Reply:
Actually, it's preferable to use `including` as asserting the whole object is more brittle to API changes.

Reply:
@morgante In this particular case, wouldn't the API including additional configs within `sysctls` be an error? My reasoning for suggesting whole object for `sysctls` was in case of any bug being introduced in the logic where we merge `all` and nodepool specific config.

Reply:
It could be an API error, but it could also be a new default that we haven't included. In either case we couldn't fix it in the module so I'd prefer to avoid brittle logic.